documentation/getting-started/README.md

```diff
@@ -22,85 +22,79 @@ You can install TestCafe Studio on MacOS, Windows or Linux. See [Installation](.
+## Add an Assertion to the Test
 
-## Add New Actions to the Test
+TestCafe Studio provides a set of [assertions](../guides/record-tests/test-actions/assertions.md) to perform verification actions in tests. You can add assertions during or after recording.
 
-You can continue the recording to add more steps to the test. The [Test Editor](../user-interface/test-editor.md) also contains an **Actions** panel with specific actions that can be added to the test during and after recording.
+This step adds an [Equals](../guides/record-tests/test-actions/assertions.md#equals) assertion to the test. The assertion should check if the **Thank you** page displays the name you entered as **Your name**. Follow the steps to add the assertion:
 
-This step adds an assertion that checks if the **Thank you** page displays the name you entered as **Your name**. You can add the assertion while recording to select which object to verify on the tested page.
+1. Click the ![Record test actions](../../images/getting-started/record-test-icon.png) **Record test actions** button on the **Test Editor**'s toolbar to continue recording the test. TestCafe Studio replays the previous test actions and waits for new ones.
 
-To continue recording the test, click the ![Record test actions](../../images/getting-started/record-test-icon.png) **Record test actions** button on the toolbar.
-TestCafe Studio replays the previous actions and waits for new ones.
-Click the button ![Equals assertion](../../images/getting-started/assertion-eql-icon.svg) in the **Actions** panel's **Assertions** section.
-The assertion is added to the end of the test.
+2. Click the ![Equals assertion](../../images/getting-started/assertion-eql-icon.svg) button in the **Actions** panel's **Assertions** section to add the assertion to the test. TestCafe Studio adds the assertion to the end of the test and allows you to specify the assertion's parameters.
 
-Next, specify the assertion parameters in the **Test Editor**:
-
-1. Click the button ![Pick a target element](../../images/getting-started/element-picker-icon.png) next to the **Actual** field to select an element whose property you are going to check. This displays the tested webpage where you can select the target element. Hover over the page header that says *'Thank you, ...'* to show the **Element Selector** popup that contains a CSS selector identifying the page header.
+3. Select an element whose property you are going to check. To do this, click the ![Pick a target element](../../images/getting-started/element-picker-icon.png) button next to the **Actual** field. This displays the tested webpage where you can select the target element. Hover the mouse cursor over the page header that says *'Thank you, ...'*. The selector popup displays the [Element Selector](../guides/record-tests/test-actions/on-page-actions/action-parameters.md#element-selector) that identifies the page header.
```

Comment:
Can we get rid of `This displays the tested webpage where you can select the target element.`? For me it can be lit confusing.